src/commands/go-cache-save.yaml

```diff
@@ -1,5 +1,10 @@
+parameters:
+  lockfile:
+    type: "string"
+    default: "go.sum"
 steps:
   - save_cache:
-      key: go-build-go-mod-v1-{{ checksum "go.sum" }}
+      key: go-build-go-mod-v1-{{ checksum "<< parameters.lockfile >>" }}
       paths:
+        - "/go/pkg/dep"
```

Comment:
Can we create a separate legacy commands for saving/restoring cache? I'd like to make the cleanup easier in the future.

Reply:
I tried to reduce code duplication using a default argument and a single command. Happy to add separate legacy commands though.

Reply:
I see what you mean and I agree with that in general but in this case when we consider those jobs temporary I'd like to make future removal easier.

Reply:
Done.